elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -112,18 +120,12 @@ async def attach_cve_flaws(runtime, advisory_id, noop, advisory, bug_tracker):
     advisory = get_updated_advisory_rhsa(runtime.logger, cve_boilerplate, advisory, first_fix_flaw_bugs)
     if not noop:
         runtime.logger.info("Updating advisory details %s", advisory_id)
+        # this removes all attached jira bugs :/
         advisory.commit()
```

Comment:
advisory.commit removes all jira bugs. I'm looking at errata-tool source code, and this certainly is the culprit

Reply:
https://github.com/red-hat-storage/errata-tool/pull/231